cmd/okctl/venv.go

```diff
@@ -30,9 +36,9 @@ export OKCTL_SHELL=/bin/bash
 okctl venv myenv
 `
 
-// nolint: funlen
+// nolint: gocyclo, funlen, gocognit
```

Comment:
Is there any way to not ignore so many lint hints? (Split up command or similar?)
(I am a bit late with reviews, so if you are already ready to merge, feel free to ignore this)

Reply:
Yeah, good points. I did so because I've till now been much less strict about the code in this package, because of the more nested nature of the cobra commands. But, that is not a good excuse really, so I have now extracted some functions to remove all the lint ignores.